sdk/php/src/ValueObject/Type.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dagger\ValueObject;
+
+use Dagger\Client\AbstractScalar;
+use Dagger\Client\IdAble;
+use Dagger\Exception\MissingSubtype;
+use Dagger\Exception\UnsupportedType;
+use Dagger\TypeDefKind;
+use DomainException;
+use ReflectionClass;
+use ReflectionNamedType;
+use ReflectionType;
+use RuntimeException;
+
+//@TODO support intersection types
```

Comment:
You mean "union" types? Those don't exist in Dagger.

Reply:
Thank you, fixed.

Reply:
I meant that "union" types don't exist in Dagger 🙂

Reply:
Ah! I apologise, I misunderstood.

I assume Dagger does not support "union" or "intersection" types?

I have removed this TODO comment entirely.

Reply:
Can you remove this comment to avoid confusion please?

Reply:
The comment is now removed. :+1: